src/components/Tabs/index.tsx

```diff
@@ -17,7 +17,7 @@ const Tabs = ({ tabs, onSelect, activeTab }: TabsProps) => (
                     activeTab === tab ? styles.active : ''
                 }`}
             >
-                {tab}
+                {tab.split('_').join('')}
```

Comment:
It is not working as shown in the image right ?

Reply:
Yes and there will be no space between because these are tabs